components/mpas-seaice/bld/namelist_files/namelist_defaults_mpassi.xml

```diff
@@ -512,7 +512,7 @@
 <config_AM_ridgingDiagnostics_write_on_startup>true</config_AM_ridgingDiagnostics_write_on_startup>
 
 <!-- AM_conservationCheck -->
-<config_AM_conservationCheck_enable>false</config_AM_conservationCheck_enable>
+<config_AM_conservationCheck_enable>true</config_AM_conservationCheck_enable>
```

Comment:
This should be a separate PR, I think.

Reply:
Thanks @xylar -- I agree, especially since mpas-seaice is also used for F-cases where its performance is already an issue